composer/models/resnet_cifar10/model.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021 MosaicML. All Rights Reserved.
+
+"""ResNet models for CIFAR10 extending :class:`.ComposerClassifier`."""
+
+from typing import List, Optional
+
+from composer.models.base import ComposerClassifier
+from composer.models.model_hparams import Initializer
+from composer.models.resnet_cifar10.resnets import CIFARResNet, ResNet9
+
+__all__ = ["CIFAR10ResNet"]
+
+
+class CIFAR10ResNet(ComposerClassifier):
```

Comment:
The nomenclature seems a bit confusing: there exists `CIFAR10ResNet` (a `ComposerClassifier`), `CIFARResNet` (an `nn.Module`), and `resnet_cifar10` (a registry/config key). What do you think about renaming to `CIFAR10ResNet` -> `ComposerCIFARResNet` and `resnet_cifar10` -> `cifar_resnet` (or ResNetCIFAR/resnet_cifar)?